usr/bin/tcemirror.sh

```diff
@@ -12,6 +12,15 @@ if [ ! -s "$TMP" ]; then
    exit 1
 fi
 select "Tiny Core Linux - Mirror Selection" "$TMP"
-ANS="$(cat  /tmp/select.ans)"
+ANS="$(cat /tmp/select.ans)"
 rm "$TMP"
+
+case "$ANS" in
+https*)
+   if ! [ -f /usr/local/tce.installed/ca-certificates ]; then
+      echo "https mirrors require the ca-certificates.tcz extension to be installed"
+      exit 1
+   fi
```

Comment:
The way its built for TC, busybox wget depends on openssl first.  Openssl is what needs certificates to function properly.

Reply:
With http, you do not need OpenSSL either.   If someone wants to use an https mirror, they would both,  openssl.tcz and ca-certificates.tcz

Reply:
I've now got this with checks for both extensions.